blockchain/blockchain-wallet-v4-frontend
# Feat (DEX): Onboarding

## Description (optional)
Dex onboarding slides
- Flow
- Use constellation
- Updated copies
- Updated images

## Testing Steps (optional)
- On staging you should see DEX on the main navigation bar
- Click on it
- First onboarding step should appear
- Go through all the steps
- Make sure Swap form is open after you click Start button
